src/components/FisheriesReport.vue

```diff
@@ -1,88 +1,131 @@
 <template>
   <div class="report">
-    <h1>{{ selectedGroup }} in {{ selectedRegion }}</h1>
-    <div id="report" v-if="groupedFisheries[selectedRegion] != undefined">
-      <button @click="reset">Back to map</button>
-      <div
-        v-for="fishery in groupedFisheries[selectedRegion][selectedGroup]"
-        :key="fishery"
-      >
+    <h1>{{ groupDict[selectedGroup] }} in {{ regionDict[selectedRegion] }}</h1>
```

Comment:
Just for semantics, I think something like `<h3>` may be better here.  (h1 is a bit "big")